src/stc/ScintillaWX.cpp

```diff
@@ -206,6 +207,8 @@ static wxTextFileType wxConvertEOLMode(int scintillaMode)
 
 
 ScintillaWX::ScintillaWX(wxStyledTextCtrl* win) {
+    GetLexerCount();
```

Comment:
It would be nice to have a comment explaining why do we call this function and don't use its result (I guess it's some kind of lexer initialization?).

Reply:
I mentioned it in the commit message. There is a bug in Lexilla 5.0.2 where it doesn't initialize the lexers. It is fixed in 5.0.3  but this version has major architecture changes so I can't/won't update to that version.

I think I'll remove this and apply the Lexilla fix instead.

Reply:
Sorry for missing this and thanks for the explanation!